sda-commons-server-mongo-testing/src/main/java/org/sdase/commons/server/mongo/testing/MongoDbRule.java

```diff
@@ -68,6 +70,12 @@
  */
 public class MongoDbRule extends ExternalResource {
   private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private static final RetryRegistry downloadMongoRetryRegistry =
+      RetryRegistry.of(
+          RetryConfig.custom()
+              .maxAttempts(10)
+              .intervalFunction(IntervalFunction.ofRandomized())
+              .build());
```

Comment:
Oh, that's a nice approach! Much better than our self-implemented retry 👍

Reply:
Has the advantage that it doesn't count into the coverage 😆